yarn.lock

```diff
@@ -1009,6 +1009,34 @@
     "@types/istanbul-reports" "^1.1.1"
     "@types/yargs" "^13.0.0"
 
+"@nodelib/fs.scandir@2.1.3":
```

Comment:
@jorgeorpinel what happened to this file?

Reply:
This must have been added when I installed `lint-staged` for the change explained in https://github.com/iterative/dvc.org/pull/776#pullrequestreview-312194024 (per https://github.com/iterative/dvc.org/issues/716#issuecomment-549939176).

Reply:
Does that answer your question?

Reply:
looks like too many changes just to this file for a single library. Also, is there a split between dev/test libraries and production?

Reply:
```console
$ yarn why @nodelib/fs.scandir 
yarn why v1.19.1
[1/4] 🤔  Why do we have the module "@nodelib/fs.scandir"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@nodelib/fs.scandir@2.1.3"
info Reasons this module exists
   - "lint-staged#del#globby#fast-glob#@nodelib#fs.walk" depends on it
   - Hoisted from "lint-staged#del#globby#fast-glob#@nodelib#fs.walk#@nodelib#fs.scandir"
info Disk size without dependencies: "108KB"
info Disk size with unique dependencies: "200KB"
info Disk size with transitive dependencies: "200KB"
info Number of shared dependencies: 2
✨  Done in 1.05s.
```

> is there a split between dev/test libraries and production?

Yes, `lint-staged` is in `devDependencies`.

Reply:
Continued in https://github.com/iterative/dvc.org/pull/776#pullrequestreview-312802656.